test/e2e/rukpakctl_test.go

```diff
@@ -17,7 +17,7 @@ import (
 )
 
 const (
-	rukpakctlcmd = "../../bin/linux/rukpakctl "
+	rukpakctlcmd = "../../bin/rukpakctl "
```

Comment:
It would be nice to give the testing suite a base directory vs. constructing the relatively path in the future.

Reply:
Does our e2e make task always ensure that the `../../bin/rukpakctl` binary is re-built?

Reply:
Is the rukpakctl built there, too?

Reply:
It depends on how the Makefile is invoked.
- With `make build-container`, we do `export GOOS=linux` because we know that containers will only be running on linux systems. We also build rukpakctl into the rukpak container so that folks can consume it that way if they want to. In this case, we put these binaries in `bin/linux`
- With `make build` we leave GOOS alone and we put binaries in `bin`

I think what we probably need to do in this case is figure out how to:
1. build rukpakctl for the local environment at `./bin/rukpakctl`
2. build the rukpak container (as usual)

IIRC, there was some makefile issue with setting GOOS=linux _just_ in the context of the `build-container` target such that other targets would _not_ inherit that setting.